### DISCUSSION TITLE
Can I remove the code in configuration.yaml?

### DISCUSSION THREAD

**Discussion by snowboardjoe** [Pirate-Weather/pirateweather#113] [Q&A] [answered]
Been using this for a long time in HA. I noticed in the documentation that the yaml code will soon be deprecated as the HACS is saving this data now. Is it ok to remove this from the configuration.yaml at this time? Any way I can verify the setting have been saved and safe to remove? I'm on the latest version of HA, HACS and PrivateWeather.

**Comment by alexander0042** [comment, marked as answer]
Hi, 

Sorry I never replied to this! Short answer is yes, there's a discrete routine that imports the config in, and after that you're ok to remove
